app/src/main/javascript/bundles/experiment-page/src/TSnePlotViewRoute.js

```diff
@@ -308,7 +317,12 @@ TSnePlotViewRoute.propTypes = {
     value: PropTypes.string.isRequired
   }).isRequired).isRequired,
   selectedK: PropTypes.number,
-  anatomogram: PropTypes.object.isRequired
+  anatomogram: PropTypes.object.isRequired,
+  initialCellTypeValues: PropTypes.array
+}
+
+TSnePlotViewRoute.defaultProps = {
```

Comment:
Do we need an equivalent here for UMAP or this is just an unfortunate name being kept for this object and this applies as well to the UMAP dimred with nothing in addition needed?

Reply:
Its just the component name we have which we didn't change. We do not need anything additional for UMAP.